cirq-superstaq/cirq_superstaq/ops/qudit_gates.py

```diff
@@ -236,6 +236,130 @@ def __repr__(self) -> str:
         )
 
 
+class VirtualZPowGate(cirq.EigenGate):
+    """Applies a phase rotation between two successive energy levels of a qudit."""
+
+    def __init__(
+        self,
+        dimension: int = 2,
+        level: int = 1,
+        exponent: cirq.TParamVal = 1.0,
+        global_shift: float = 0.0,
+    ) -> None:
+        """Initializes as `VirtualZPowGate`.
+
+        Args:
+            dimension: The qudit dimension on which this gate will act.
```

Comment:
Can you add something explaining how the acronym / abbrev. relates to dimension and level? E.g. VZ3_12 not really intuitive that it means VirtualZPowGate(dimension=3, level=2)

Reply:
ended up changing all the aliases to e.g. `QutritVZ2` (for dimension 3, level 2) - lmk if you think they're any more clear